pkg/api/operators/operators_test.go

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+package operators
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/controller"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutil"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutilcontroller"
+	"github.com/verrazzano/verrazzano-operator/pkg/types"
+	v1wlsopr "github.com/verrazzano/verrazzano-wko-operator/pkg/apis/verrazzano/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+const wlsNamespace, wlsOperatorName, wlsBindingName = "test", "wls-operator", "mytestbinding"
+
+// TestInit ensures that a set of operators can be obtained from the provided set of listers.
+func TestInit(t *testing.T) {
+	// Get a set of clusters
+	Operators = nil
+	clusters := testutil.GetTestClusters()
+	var clients kubernetes.Interface = fake.NewSimpleClientset()
+	modelBindingPairs := map[string]*types.ModelBindingPair{
+		"test-pair-1": testutil.GetModelBindingPair(),
+	}
+	type args struct {
+		listers controller.Listers
+	}
+	tests := []struct {
+		name    string
+		listers controller.Listers
+	}{
+		{name: "testInit",
+			listers: testutilcontroller.NewControllerListers(&clients, clusters, &modelBindingPairs)},
+	}
+	for _, tt := range tests {
+		// add a WLSOperator to one of the clusters
+		t.Run(tt.name, func(t *testing.T) {
+			// append the WLS operator
```

Comment:
also, can I put in a word in favor of the "given/when/then" style of comments which delineate between set up, the call to the method under test, and the assertions? The content of the comments is largely useless since it gets outdated, but the delineation makes it a bit easier to read IMO